src/components/Articles/Articles.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-card flat id="articles">
+    <confirm-dialog ref="confirm" attach="#app" />
+
+    <div class="define-article-header">
+      <v-icon color="app-dk-blue">mdi-handshake</v-icon>
+      <label class="define-article-title">Articles</label>
+    </div>
+
+    <div class="section-container">
+      <company-provisions
+        class="sub-section"
+        :provisionsRemoved="getProvisionsRemoved"
+        @companyProvisionsChanged="updateProvisionsRemoved($event)"
+        @haveChanges="emitHaveChanges($event)"/>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Mixins, Watch } from 'vue-property-decorator'
+import { CommonMixin } from '@/mixins'
+import { ConfirmDialog } from '@/components/dialogs'
+import CompanyProvisions from './CompanyProvisions.vue'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF } from '@/interfaces'
+
+@Component({
+  components: {
+    ConfirmDialog,
+    CompanyProvisions
+  }
+})
+export default class Articles extends Mixins(CommonMixin) {
+  // whether components have changes
+  private companyProvisionsChanges = false
+
+  @Getter getProvisionsRemoved!: boolean
+
+  // Setters
+  @Action setProvisionsRemoved!: ActionBindingIF
+
+  private updateProvisionsRemoved (provisionsRemoved: boolean): void {
+    this.setProvisionsRemoved(provisionsRemoved)
+  }
+
+  private setDataChanges (): void {
+    const haveChanges: boolean = this.companyProvisionsChanges
+    this.emitHaveChanges(haveChanges)
+  }
+
+  /** Emits Have Changes event. */
+  @Emit('haveChanges')
+  private emitHaveChanges (haveChanges: boolean): void {}
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+.section-container {
+  padding: 1.25rem 1rem;
+
+  .sub-section {
+    margin-top: 1.5rem;
+  }
+}
+
+.define-article-header {
+  display: flex;
+  background-color: $BCgovBlue5O;
+  padding: 1.25rem;
+}
+
+.define-article-title {
+  padding-left: 0.5rem;
```

Comment:
I noticed inconsistency between these titles across the various components.
- they should all be bold (can use class="font-weight-bold" instead of `<strong>`)
- they should all have 0.5 rem left padding (can use class="pl-1" instead)

Could you please check the other titles and format them all similarly?

My preference is to keep specifying an id or classname for them (for testing selector purposes) but to have the bold and padding classes right in the HTML.

Reply:
Will check. Thanks.

Reply:
Ok. I'm going to use a css class but it's pl-1 in this case. According to Tracey we should keep aligned to YourCompany component and I'm doing this.

Reply:
Yes, pl-1 sounds right,

Do all titles have the same style now?

Reply:
Sorry I meant: it's **not** pl-1 in this case.

Reply:
I would like to see all titles with the same styles (according to the ux specs). Can you confirm that all titles have the same classes? `class="font-weight-bold pl-2"` or whatever?

Reply:
Most of them have a custom classes and some use font-weight-bold instead. I'm going to use font-weight-bold too.